pkg/store/cache/caching_bucket_factory.go

```diff
@@ -30,6 +38,41 @@ type CachingBucketWithBackendConfig struct {
 	CachingBucketConfig CachingBucketConfig `yaml:"caching_config"`
 }
 
+type CachingBucketConfig struct {
+	// Basic unit used to cache chunks.
+	ChunkSubrangeSize int64 `yaml:"chunk_subrange_size"`
+
+	// Maximum number of GetRange requests issued by this bucket for single GetRange call. Zero or negative value = unlimited.
+	MaxChunksGetRangeRequests int `yaml:"max_chunks_get_range_requests"`
+
+	// TTLs for various cache items.
+	ChunkObjectSizeTTL time.Duration `yaml:"chunk_object_size_ttl"`
+	ChunkSubrangeTTL   time.Duration `yaml:"chunk_subrange_ttl"`
+
+	// How long to cache result of Iter call in root directory.
+	RootIterTTL time.Duration `yaml:"root_iter_ttl"`
```

Comment:
From the user perspective this is the "blocks iteration". The fact blocks are stored in the root is more an implementation detail (and we may soon support custom prefix support to the bucket). For this reason I would call it `blocks_iter_ttl`.